server/views/pages/barcode/generate-barcode-image.njk

```diff
@@ -64,11 +64,10 @@
       </div>
     {% endfor %}
 
-    <div class="govuk-grid-row govuk-!-margin-top-6">
+    <div class="govuk-grid-row govuk-!-margin-bottom-6">
       <div class="govuk-grid-column-full">
-        <p class="govuk-!-margin-top-6 govuk-!-margin-bottom-6">
```

Comment:
Liora has confirmed that they no longer want the link to the survey herre, but do want the link to start again instead